### PR TITLE
2.5.1 -- Slash Extensions & Refactor : Refactor application commands to have instance attributes instead of properties

### PR DESCRIPTION
**Pull request by FlamptX**
1. Changed private callback properties into public attributes in ApplicationCommand class and SlashCommandChild class.
2. A few very small PEP 8 fixes that I couldn't resist to leave like that :)

From the 2.5.1 todo list, card link; https://github.com/diskord-dev/diskord/projects/2#card-69995521

### CODE REVIEW

File: diskord/application_commands.py [hunk 1]
```diff
@@ -551,24 +551,27 @@ class ApplicationCommand:
 
         For more info on checks and how to register them, See :func:`~ext.commands.check` 
         documentation as these checks actually come from there.
+
+    callback: Callable[..., Any]
+        Callable[..., Any]: Returns the command's callback function.
```
Comment (nerdguyahmad) [inline diskord/application_commands.py:556]:
```suggestion
        the callback function of this command.
```

File: diskord/application_commands.py [hunk 2]
```diff
@@ -994,6 +993,9 @@ class SlashCommandChild(Option):
     This class is not meant to be used in general and is here for documentation-purposes only.
     For general use, Use the subclasses of this class like  :class:`SlashCommandGroup` and
     :class:`SlashSubCommand`.
+
+    callback
+        Callable: The callback function for this child.
```
Comment (nerdguyahmad) [inline diskord/application_commands.py:998]:
```suggestion
    callback: Callable[..., Any]
        The callback function for this child.
```

**Review by nerdguyahmad — CHANGES_REQUESTED** [review]
Some improvements should be done to docstring.

**Comment by nerdguyahmad** [comment]
Thanks.

**Comment by FlamptX** [comment]
> Thanks.

You got it :)

**Pull request merged**